client/src/datafiles/layouts/FileListingLayout.tsx

```diff
@@ -42,8 +42,13 @@ export const FileListingLayout: React.FC = () => {
     'designsafe.storage.frontera.work',
   ].includes(system);
 
+  const isUserWorkSystem = ['cloud.data'].includes(system);
+
   const redirectHome =
     user?.username && !path && api === 'tapis' && isUserHomeSystem;
+
+  const redirectWork =
```

Comment:
Will this only redirect if the path starts with /work?

Reply:
It'll redirect as long as there isn't a specified path.